src/dqe_trans2.erl

```diff
@@ -20,7 +20,7 @@ start({_Start, _Count}, State) ->
     {ok, State}.
 
 emit(_Child, {realized, {Data, Resolution}}, State = #state{op = Op, arg = Arg}) ->
```

Comment:
Line 22 is too long: emit(_Child, {realized, {Data, Resolution}}, State = #state{op = Op, arg = Arg}) ->.